namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-frontend-staging/resources/ecr.tf

```diff
@@ -0,0 +1,26 @@
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=7.0.0"
+
+  # Repository configuration
+  repo_name = var.namespace
+
+  # OpenID Connect configuration
+  oidc_providers      = ["github"]
+  github_repositories = ["laa-crime-equinity-historical-data-frontend"]
```

Comment:
This repository is already mentioned in the laa-crime-equinity-historical-data-frontend-dev environment. Use github_environment so both credentials dont get overwritten